public/manifest.json

```diff
@@ -4,7 +4,7 @@
   "Scope": "/news/",
   "background_color": "#b80000",
   "display": "standalone",
-  "start_url": "/news/",
+  "start_url": "/news/articles/c0000000025o",
```

Comment:
Should this be a URL for a specific article?

Would usually be something like an index link here.